minitask3/akhmad/follower.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+import rospy
+import cv2
+import cv_bridge
+import numpy
+
+from geometry_msgs.msg import Twist
+from sensor_msgs.msg import Image, LaserScan
+
+LOWER_GREEN = numpy.array([60, 100, 50])
+UPPER_GREEN = numpy.array([60, 255, 255])
+
+SAMPLE_WINDOW = 20  # Degrees
+
+TURN_MAX_SPEED = 0.2
```

Comment:
its normally worth doing a min speed too, so the robot is going straight forward when it can.